src/main/plugins/org.dita.base/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -697,12 +705,12 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
       <xsl:when test="empty($targetElement)"/>
       <!--otherwise try pulling shortdesc from target-->
       <xsl:otherwise>
-        <xsl:variable name="shortdesc">
+        <xsl:variable name="shortdesc" as="node()*">
```

Comment:
Without typing this variable will create a new document node as the root and we don't want that.